main/externalData/coingecko/index.ts

```diff
@@ -22,7 +22,7 @@ export interface Coin {
   symbol: string
   name: string
   asset_platform_id: string
-  platforms?: {}
+  platforms?: Record<string, never> | { [platformId: string]: string }
```

Comment:
this typing doesn't seem quite right. why would the value be `never`?

Reply:
`Record<string, never>` is the recommended way to represent an empty object type, given that `{}` means any non-nullish value

https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492

Reply:
I was gonna recommend we can just use `Record<string, string>` but I don't think we even use this coingecko code at all anymore do we? can probably just trash the entire module(s)

Reply:
is gone